src/components/shared/searchBar/searchBar.css

```diff
@@ -22,3 +22,14 @@
     height: 3px;
   }
 }
+
+.searchInput + span {
```

Comment:
💅 According to our CSS guide, we shouldn't use generic tags (like span or div) for CSS selectors.